.evergreen/serverless/create-instance.sh

```diff
@@ -113,6 +114,12 @@ SINGLE_ATLASPROXY_SERVERLESS_URI: "$SERVERLESS_URI"
 MULTI_ATLASPROXY_SERVERLESS_URI: "$SERVERLESS_URI"
 SERVERLESS_MONGODB_VERSION: "$SERVERLESS_MONGODB_VERSION"
 EOF
+
+        if [ "$SERVERLESS_SKIP_CRYPT" != "OFF" ]; then
```

Comment:
given a default value is `true`, should we use `false` here instead `OFF`?

Reply:
I've updated the comment to note the default value as `ON`. I used `ON` and `OFF` for consistency with the `LOADBALANCED` variable.